back/src/test/java/com/ubikloadpack/jmeter/ulp/observability/log/SampleLogTest.java

```diff
@@ -0,0 +1,109 @@
+package com.ubikloadpack.jmeter.ulp.observability.log;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Date;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import com.ubikloadpack.jmeter.ulp.observability.log.SampleLog;
+import com.ubikloadpack.jmeter.ulp.observability.log.SampleLoggerTest;
+import com.ubikloadpack.jmeter.ulp.observability.metric.ResponseResult;
+import com.ubikloadpack.jmeter.ulp.observability.registry.MicrometerRegistry;
+import com.ubikloadpack.jmeter.ulp.observability.util.Util;
+
+import io.micrometer.core.instrument.distribution.ValueAtPercentile;
+
+public class SampleLogTest {
+	
+	@Test
+	public void testToOpenMetricsString() {

```

Comment:
Name your test like this:
- no test in name (we know it's a test method)
- whenSomethingExpectResult